Integrations/integration-VxStream.yml

```diff
@@ -284,6 +288,29 @@ script:
         return res;
     }
 
+    function detonateFile(entryId, environmentId, delay, timeout) {
+
+        var hash = submitFile(entryId, environmentId).substring(10);
+
+        delayTime = parseInt(delay);
+        timeOut = parseInt(timeout);
+        var waitTime = delayTime;
+
+        wait(delayTime);
+
+        while (waitTime<timeOut) {
+            res = scan(hash);
+            if (res.response[0]) {
```

Comment:
I would check also if res.length > 0, so you won't get out of range exception/ 
`if (res.response.length > 0 && res.response[0])`